plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/ImportNativeAppEngineStandardProjectTest.java

```diff
@@ -65,6 +65,7 @@ public void importAppEngineStandardJava7_from1_3_1() throws IOException, CoreExc
 
     updateOldContainers();
 
+    ProjectUtils.waitForProjects(project);
```

Comment:
This looks significant enough to go into a focused PR of its own rather than a PR of assorted minor cleanups.

Reply:
Backing out and created #2512.